packages/monitoring-system/metrics-dashboard/scripts/deploy.sh

```diff
@@ -0,0 +1,17 @@
+# build and deploy the dashboard from a fresh clone
+
+set -eo pipefail  # fail-fast and don't mask errors
+
+# check token was passed in args
+if [[ $# -ne 1 ]]
+then
+  echo "Usage $0 <path-to-token-file>"
+  exit 1
+fi
+
+# compile
+npm install
+npm run compile
+
+# deploy to Firebase
+firebase deploy --token '$(cat $1)'
```

Comment:
I think you want double-quotes or it'll use the raw `$(cat $1)` as a string

Reply:
yeah I wasn't sure about that. I'll use double!